pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "scipion-em-smartscope"
+dynamic = ["version"]
+dependencies = [
```

Comment:
There is a way to use requirements.txt

dynamic = ["version", "dependencies"]

And later:

[tool.setuptools.dynamic]
version = {attr = "smartscope.__version__"}
dependencies = {file = ["requirements.txt"]}